src/Controls/tests/UITests/Tests/Issues/Issue19509.cs

```diff
@@ -0,0 +1,31 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	class Issue19509 : _IssuesUITest
+	{
+		public Issue19509(TestDevice device) : base(device)
+		{
+		}
+
+		public override string Issue => "Entry TextColor property not working when the Text value is bound after some time";
+
+		[Test]
+		public void Issue19509Test()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Android, TestDevice.Mac, TestDevice.Windows });
+
+			App.WaitForElement("WaitForStubControl");
+
+			// 1. Click a button to update the text
+			App.Click("button");
+
+			// 2. Verify that the Entry bounded TextColor is correct (Green).
+			var color = App.FindElement("").GetText();
+			Assert.AreEqual("[Color: Red=0, Green=0.5019608, Blue=0, Alpha=1]", color);
```

Comment:
TextColor won't propagate back up if it's changed unexpectedly on the platform

![image](https://github.com/dotnet/maui/assets/5375137/3881a990-aba5-4caa-b6a0-22ad2b54546e)

I think we'll need to test this through screen shot comparisons

Reply:
That's a good idea